packages/arb-token-bridge-ui/src/components/TransferPanel/CctpTabContent.tsx

```diff
@@ -0,0 +1,45 @@
+import { PropsWithChildren } from 'react'
+import { CCTP_DOCUMENTATION } from '../../constants'
+import { useCCTPIsBlocked } from '../../hooks/CCTP/useCCTPIsBlocked'
+import { ExternalLink } from '../common/ExternalLink'
+
+export const CctpTabContent = ({
+  toNetworkName,
+  children
+}: PropsWithChildren<{
+  toNetworkName: string
+}>) => {
+  const { data: isCctpBlocked, isLoading: isLoadingIsCctpBlocked } =
+    useCCTPIsBlocked()
+
+  if (isLoadingIsCctpBlocked || isCctpBlocked) {
+    return (
+      <>
+        Access to Circle&apos;s bridge is restricted in certain jurisdictions.
```

Comment:
other tabs use `font-light` (300) whereas here it's using 400

Reply:
Good catch, I now wrap it with a `p` tag